app/controlplane/pkg/data/ent/migrate/migrations/20241107213854.sql

```diff
@@ -0,0 +1,12 @@
+-- we mark the latest version of each project as prerelease and unset the rest
```

Comment:
I don't understand the query. If want we want is to mark all existing pre-releases as released, the query I don't think that does that, I guess it shall be something like this:

```sql
WITH ranked_versions AS (SELECT id,
                                ROW_NUMBER() OVER (PARTITION BY project_id ORDER BY created_at DESC) as rn
                         FROM project_versions
                         WHERE prerelease = true)
UPDATE project_versions
SET prerelease = false
WHERE (id IN (SELECT id
              FROM ranked_versions
              WHERE rn = 1));
```